polkadot/xcm/src/v3/junction.rs

```diff
@@ -118,8 +114,6 @@ impl From<NewNetworkId> for NetworkId {
 			ByFork { block_number, block_hash } => Self::ByFork { block_number, block_hash },
 			Polkadot => Self::Polkadot,
 			Kusama => Self::Kusama,
-			Westend => Self::Westend,
-			Rococo => Self::Rococo,
```

Comment:
you cannot change this struct for a live version - this struct is part of the XCM ABI, it influences the encoding, changing it like this will break all chains using XCM